src/Compute/custom/Strategies/ComputeRp/IResource.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.Azure.Commands.Compute.Strategies.ComputeRp
+{
+    public interface IResource
```

Comment:
@markcowl should this interface also have an `Id` property?

Reply:
Not for this usage.